qaframework-bdd-tests/src/test/resources/features/refapp-2.x/13-appointment/appointments.feature

```diff
@@ -1 +1,70 @@
-Feature: Appointments Management
\ No newline at end of file
+Feature: Appointment Management
+
+	Background:
+		Given a user clicks on Request Appointment link from the Patient dashboard
+		Then the system loads Request Appointment page
+		And a user fills in the request details
+		And a user clicks save button
+		Then the system adds appointment request into the appointment requests table
+		
+	@selenium
+	@appointment
+	Scenario: Create Appointment Block, Edit Appointment Block, Add Appointment and Delete Appointment Block
+		# User story: Create Appointment Block
+		When a user clicks Appointment scheduling app from the home page
+		Then the system loads the appointment scheduling page
+		And a user clicks on Manage Provider Schedules app
+		And the system loads Manage Provider Schedules page
```

Comment:
These workflow are too many  which looks a bit not clear. Try to make sure to minimize the user stories as much as possible to make them meanfull. Suppose this workflow land on you after 10 years ago what would you think. Make sure you refactor them to reduce on the user stories but keeping them appropriate, readable and making meaning.

Reply:
@sherrif10 actually those work flows build on top of each other. One must have created an appointment request as the initial step. that is why I have put that as the background. Then from there, the user has to create an appointment block on which the user builds to create an appointment. and then the edit and delete also definitely have to build on top of a created appointment block. So that is why the four workflows are bound under one scenario because they have to rely on a created appointment block.

Reply:
If I am to separate them, then it means that in every scenario I have to create an appointment block. Would that be a better implementation of this @sherrif10 . Let me know so I can change to that.